test/langtools/tools/javac/patterns/SwitchErrors-no-preview.out

```diff
@@ -0,0 +1,40 @@
+SwitchErrors.java:35:31: compiler.err.preview.feature.disabled.plural: (compiler.misc.feature.pattern.switch)
```

Comment:
I don't get why you want to test what happens when the `--enable-preview + source` options are not passed

Reply:
I apologize for a belated answer. The idea to test what happens when preview is disabled is that the error recovery messages are different. The error message won't suggest to use a pattern when pattern matching in switch is not enabled. That may or may not be the ideal behavior, but it seemed prudent to verify that behavior. I can split it into a separate test case, to make that more obvious, if desired.